flapp-webapp/src/components/FPO.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div
+    class="fill-height-lg survey-container contentcontainer codecontainer"
+    id="fpocomponent"
+  >
+    <h2>FPO form</h2>
+    <b-container class="fill-body">
+      <survey :survey="survey"></survey>
+      <!--div id="surveyResult"></div-->
+    </b-container>
+  </div>
+</template>
+
+<script>
+import * as SurveyVue from "survey-vue";
+import surveyPrimaryJSON from "../assets/survey-information.json";
+import { addQuestionTypes } from "./question-types.ts";
+
+SurveyVue.StylesManager.applyTheme("bcgov");
+addQuestionTypes(SurveyVue);
+
+/* survey.onComplete.add(function(result) {
+  document.querySelector("#surveyResult").textContent =
+    "Result JSON:\n" + JSON.stringify(result.data, null, 3);
+}); */
+
+export default {
+  name: "FPO",
+  data() {
+    var survey = new SurveyVue.Model(surveyPrimaryJSON);
+    //console.log("surveyPrimaryJSON = " + JSON.stringify(surveyPrimaryJSON, null, 3));
+    return { survey: survey };
+  }
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped lang="scss">
+@import "../styles/_survey.scss";
```

Comment:
`@import "../styles/survey"`, Sass can figure out the rest. That said, it should not be imported in multiple places to avoid duplicating the style rules.